src/hypothesis/internal/debug.py

```diff
@@ -42,3 +45,28 @@ def minimal(
         settings=settings,
         random=random,
     )
+
+
+HYPOTHESIS_ROOT = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), '..'))
```

Comment:
I think it doesn't matter, but I also think your way is clearer so I'll do that.